src/main/java/me/devsnox/playtime/PlayTime.kt

```diff
@@ -0,0 +1,51 @@
+package me.devsnox.playtime
+
+import me.devsnox.playtime.commands.PlayTimeCommand
+import me.devsnox.playtime.configuration.ConfigurationHandler
+import me.devsnox.playtime.listeners.PlayerListener
+import me.devsnox.playtime.playtime.PlayTimePlaceholder
+import me.devsnox.playtime.playtime.TimeManager
+import me.devsnox.playtime.utils.Messages
+import org.bukkit.Bukkit
+import org.bukkit.ChatColor
+import org.bukkit.configuration.file.YamlConfiguration
+import org.bukkit.plugin.java.JavaPlugin
+import java.io.File
+
+/**
+ * Created by Yasin Dalal
+ * Copyright (c) 2017-2019 Yasin Dalal
+ * GitHub: https://github.com/DevSnox
+ * E-Mail: yasin@dalal.ch
+ */
+class PlayTime : JavaPlugin() {
+
+    private val timeManager: TimeManager by lazy {
+        TimeManager(this, ConfigurationHandler.configurateCredentials(this))
+    }
+
+    override fun onEnable() {
+        this.saveResource("credentials.yml", false)
+        this.saveResource("messages.yml", false)
+
+        ConfigurationHandler.configurateMessages(this)
+
+        timeManager.startup()
+        registerPlaceholderAPI()
+
+        Bukkit.getPluginManager().registerEvents(PlayerListener(timeManager), this)
```

Comment:
Put the registration into the Listerner or minium use `server.pluginmanager` instead of `Bukkit.getPluginManager()`